CONTRIBUTING.md

```diff
@@ -72,7 +72,7 @@ License Agreement. This is not a copyright _assignment_; it simply gives
 Google permission to use and redistribute your contributions as part of the
 project.
 
-  - If you are an individual writing original source code and you're sure you
+  - If you are an individual writing original source code, and you're sure you
```

Comment:
Wasn‘t really needed and now sounds like an artificial pause/emphasis is set for the second part?